src/main/java/slimeknights/tconstruct/library/recipe/tinkerstation/building/ToolBuildingRecipe.java

```diff
@@ -38,7 +37,6 @@
  * This recipe is used for crafting a set of parts into a tool
  */
 
-@AllArgsConstructor
```

Comment:
If you make `layoutSlot` final, you can do a `@RequiredArgsConstructor` here. I don't think `layoutSlot` has a need to be non-final.